crates/wasm-subsystem/wasm-resolver-dynamic/src/lib.rs

```diff
@@ -7,8 +7,11 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-pub use plugin::WasmSubsystemLoader;
+//! Dynamic loader for postgres-resolver.
 
-mod plugin;
-mod wasm_execution_error;
-mod wasm_operation;
+use core_plugin_interface::interface::SubsystemLoader;
+use wasm_resolver::WasmSubsystemLoader;
+
+// See comments in `postgres-resolver-dynamic/src/lib.rs`.
```

Comment:
and here

Reply:
This one is okay. The comment is in postgres-resolver-dynamic.